save-demo/src/main/kotlin/com/saveourtool/save/demo/utils/HttpUtils.kt

```diff
@@ -0,0 +1,24 @@
+@file:Suppress("HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE")
```

Comment:
How we have three (yes, **three**, Carl!) files named `HttpUtils.kt` in our codebase.

Maybe it's time to finally merge their content?

Reply:
It seems that one of this files is located in `Kotlin Native` section while another one from `jvm` section (save-api) does not depend on `reactor`. In my third `HttpUtils.kt` file I need both `reactor` and `ktor` dependencies (as well as `spring` lol)